### ISSUE TITLE
test_scc.py fails tests with python-3.10

### ISSUE THREAD

**Issue by cyberpunkrocker-zero** [pbs/pycaption#268]
I'm trying to (re)build pycaption on Arch Linux, which recently upgraded to python-3.10 (had no issues with python-3.9), but I get the following test failure:
```
==> Starting check()...
============================= test session starts ==============================
platform linux -- Python 3.10.1, pytest-6.2.5, py-1.11.0, pluggy-0.13.1
rootdir: /home/madmax/aur/python-pycaption/src/pycaption-2.0.3
plugins: lazy-fixture-0.6.3, anyio-3.4.0
collected 221 items                                                            

tests/test_base.py .......                                               [  3%]
tests/test_dfxp.py ......................                                [ 13%]
tests/test_dfxp_conversion.py ...............................            [ 27%]
tests/test_dfxp_extras.py .......                                        [ 30%]
tests/test_functions.py .                                                [ 30%]
tests/test_geometry.py ........                                          [ 34%]
tests/test_microdvd.py ............                                      [ 39%]
tests/test_microdvd_conversion.py ..                                     [ 40%]
tests/test_sami.py ...........................                           [ 52%]
tests/test_sami_conversion.py ..........                                 [ 57%]
tests/test_scc.py E

==================================== ERRORS ====================================
___ ERROR at setup of TestSCCReader.test_reader_only_supports_unicode_input ____

request = <FixtureRequest for <Function test_reader_only_supports_unicode_input>>

    def fill(request):
        item = request._pyfuncitem
        fixturenames = getattr(item, "fixturenames", None)
        if fixturenames is None:
            fixturenames = request.fixturenames
    
        if hasattr(item, 'callspec'):
            for param, val in sorted_by_dependency(item.callspec.params, fixturenames):
                if val is not None and is_lazy_fixture(val):
                    item.callspec.params[param] = request.getfixturevalue(val.name)
                elif param not in item.funcargs:
                    item.funcargs[param] = request.getfixturevalue(param)
    
>       _fillfixtures()

/usr/lib/python3.10/site-packages/pytest_lazyfixture.py:39: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_scc.py:18: in setup_method
    self.reader = SCCReader()
pycaption/scc/__init__.py:174: in __init__
    self.buffer_dict.add_change_observer(self._flush_implicit_buffers)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = {'pop': <pycaption.scc.specialized_collections.InstructionNodeCreator object at 0x7fcd4c18f3a0>, 'paint': <pycaption.s...ct at 0x7fcd4c18ca60>, 'roll': <pycaption.scc.specialized_collections.InstructionNodeCreator object at 0x7fcd4c18eec0>}
observer = <bound method SCCReader._flush_implicit_buffers of <pycaption.scc.SCCReader object at 0x7fcd4c18cd60>>

    def add_change_observer(self, observer):
        """Receives a callable function, which it will call if the active
        element changes.
    
        The observer will receive 2 positional arguments: the old and new key
    
        :param observer: any callable that can be called with 2 positional
            arguments
        """
>       if not isinstance(observer, collections.Callable):
E       AttributeError: module 'collections' has no attribute 'Callable'

pycaption/scc/specialized_collections.py:148: AttributeError
```

**Comment by solsticedhiver** [comment]
Yes. confirmed

**Comment by ana-nichifor** [comment]
Thanks for reporting this! Fixed on PyCaption 2.0.8
